docs/en/recommended_topics/deploy/mmdeploy_guide.md

```diff
@@ -19,6 +19,14 @@ Currently our deployment kit supports on the following models and backends:
 
 Note: ncnn and other inference backends support are coming soon.
 
+## Installation
+
+Please install mmdeploy by following [this](https://github.com/open-mmlab/mmdeploy/blob/main/docs/en/get_started.md) guide.
+
```

Comment:
here should be formatted as 
:::{note}
If you install mmdeploy prebuilt package, please also clone its repository with below command to get the deployment configs and tools.

```bash
git clone https://github.com/open-mmlab/mmdeploy.git --depth=1
```

:::

Reply:
@huayuan4396 it seems this could work as well. may keep as original